src/cmd/dev.go

```diff
@@ -68,7 +80,7 @@ var devDeployCmd = &cobra.Command{
 		}
 
 		// Set dev source
-		if localBundle {
+		if isLocalBundle {
```

Comment:
Instead of having 2 `if isLocalBundle` clauses, can they be combined?

Reply:
good catch :)